packages/story-editor/src/components/form/tags/input.js

```diff
@@ -135,48 +187,112 @@ function Input({
           setIsInputFocused(false);
         },
       }),
-      [onUndo]
+      [onUndo, suggestedTerms]
     );
 
   const renderedTags = tagBuffer || tags;
+
+  const handleTagSelectedFromSuggestions = useCallback((e, selectedValue) => {
+    e.preventDefault();
+    setIsSuggestionsOpen(false);
+    dispatch({ type: ACTIONS.SUBMIT_VALUE, payload: selectedValue });
+    inputRef?.current.focus();
+  }, []);
+
+  const handleSuggestionKeyDown = useCallback(
+    (e, index, name) => {
+      const nextChild = index + 1;
+      const previousChild = index - 1;
+      if (e.key === 'ArrowDown') {
+        if (nextChild < totalSuggestions) {
+          menuRef?.current?.children?.[nextChild]?.focus();
+        }
+      }
+      if (e.key === 'ArrowUp') {
+        if (previousChild < 0) {
+          inputRef?.current.focus();
+        } else {
+          menuRef?.current?.children?.[previousChild]?.focus();
+        }
+      }
+      if (e.key === 'Enter') {
+        handleTagSelectedFromSuggestions(e, name);
+      }
+    },
+    [handleTagSelectedFromSuggestions, totalSuggestions]
+  );
+
   return (
-    <Border isInputFocused={isInputFocused}>
-      {
-        // Text input should move, relative to end, with offset
-        // this helps with natural tab order and visuals
-        // as you ArrowLeft or ArrowRight through tags
-        [
-          ...renderedTags.slice(0, renderedTags.length - offset),
-          INPUT_KEY,
-          ...renderedTags.slice(renderedTags.length - offset),
-        ].map((tag) =>
-          tag === INPUT_KEY ? (
-            <TextInput
-              {...props}
-              key={INPUT_KEY}
-              value={value}
-              onKeyDown={handleKeyDown}
-              onChange={handleChange}
-              onFocus={handleFocus}
-              onBlur={handleBlur}
-              size="4"
-            />
-          ) : (
-            <Tag key={tag} onDismiss={removeTag(tag)}>
-              {tagDisplayTransformer(tag) || tag}
-            </Tag>
+    <Border ref={containerRef} isInputFocused={isInputFocused}>
+      <InputWrapper>
+        {
+          // Text input should move, relative to end, with offset
+          // this helps with natural tab order and visuals
+          // as you ArrowLeft or ArrowRight through tags
+          [
+            ...renderedTags.slice(0, renderedTags.length - offset),
+            INPUT_KEY,
+            ...renderedTags.slice(renderedTags.length - offset),
+          ].map((tag) =>
+            tag === INPUT_KEY ? (
+              <TextInput
+                {...props}
+                key={INPUT_KEY}
+                value={value}
+                onKeyDown={handleKeyDown}
+                onChange={handleChange}
+                onFocus={handleFocus}
+                onBlur={handleBlur}
+                size="4"
+                ref={inputRef}
+                autoComplete={isSuggestionsOpen ? 'off' : 'on'}
```

Comment:
I think this is backwards
```suggestion
                autoComplete={isSuggestionsOpen ? 'on' : 'off'}
```
![backwards](https://user-images.githubusercontent.com/22185279/135694423-16d5b1a5-5904-4267-b659-7bb24fe1f98d.gif)

Reply:
this is just the browser's autocomplete attribute - it's what helps in forms when you're entering common information for the browser to help you by suggesting a value to input.  If reversed, this happens: 
<img width="312" alt="Screen Shot 2021-10-03 at 2 06 48 PM" src="https://user-images.githubusercontent.com/10720454/135771566-e96a6a0a-fcf5-487d-9f17-ec5c3c8c60c2.png">

Reply:
oh I see!

Reply:
what's the need to turn it on at all vs leaving it off the whole time?

Reply:
just thought it was nice that if you had some previous input you'd already put in that you wanted to make a tag that your browser'd pick up on it and offer it before your value's long enough for the dropdown to be visible. happy to just leave it off the whole time.